python/test/unit_distributed/test_contexts_arbmpi.py

```diff
@@ -15,10 +16,15 @@
 except ModuleNotFoundError:
     from test import options
 
+# check Arbor's configuration of mpi
```

Comment:
Remove the named tuple tests. We were interested in named tuples, if we could get our `arbor.config()` to return one, but we have settled on dictionaries.

Reply:
Done